app/components/SubagentsSidebar.tsx

```
@@ -0,0 +1,829 @@
+  const selectedById = useQuery(
+    api.subagents.getOwned,
+    selectedId ? { subagentId: selectedId } : "skip",
+  ) as ChildSummary | null | undefined;
+  const persistedSelected = selectedId ? selectedById : null;
+  const effectiveParentMessageId =
+    persistedSelected?.parent_message_id ?? resolvedParentMessageId;
+  const selectedOriginResolved = !selectedId || selectedById !== undefined;
+  const runs = useQuery(api.subagents.listForParentMessage, {
+    parentMessageId: effectiveParentMessageId,
+  }) as ChildSummary[] | undefined;
+  const [now, setNow] = useState(Date.now());
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/app/components/SubagentsSidebar.tsx:544:34
  542 |     parentMessageId: effectiveParentMessageId,
  543 |   }) as ChildSummary[] | undefined;
> 544 |   const [now, setNow] = useState(Date.now());
      |                                  ^^^^^^^^^^ Cannot call impure function
  545 |   const [canceling, setCanceling] = useState(false);
  546 |   const [cancelError, setCancelError] = useState<string | null>(null);
  547 |   const openedChildren = useRef(new Set<string>());

(react-hooks/purity)